src/pal/tools/gen-buildsys-gcc.sh

```diff
@@ -177,12 +155,10 @@ cmake \
   "-DCMAKE_LINKER=$gcc_link" \
   "-DCMAKE_NM=$gcc_nm" \
   "-DCMAKE_RANLIB=$gcc_ranlib" \
-  "-DCMAKE_OBJCOPY=$gcc_objcopy" \
```

Comment:
Why remove this? This script is also used for cross compilation to ARM. We don't want to mix x64 objcopy with arm64 objcopy.

Reply:
In Clang, we don't use `OBJCOPY` so I thought it is unused. No?

Reply:
You are right, can you also remove these checks?

CMakeLists.txt:      # Ensure that objcopy is present
CMakeLists.txt:          find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
CMakeLists.txt:          find_program(OBJCOPY objcopy)
CMakeLists.txt:        find_program(OBJCOPY objcopy)
CMakeLists.txt:          message(FATAL_ERROR "objcopy not found")

Reply:
It turned out we need to keep objcopy, and in this case, clang script is probably missing the value.